dom-manipulation/main.js

```diff
@@ -0,0 +1,29 @@
+var count = 0;
+
+var hotBtn = document.querySelector('.hot-button');
+var clickCount = document.querySelector('.click-count');
+
+hotBtn.addEventListener('click', function (event) {
+  count += 1;
+  clickCount.textContent = 'Clicks: ' + count;
+  if (count === 4) {
+    var temp = document.querySelector('.hot-button.cold');
```

Comment:
Even if you change the classes applied to it, the `hotBtn` variable will always hold the same DOM element, so you shouldn't need to query for the element again in this function, you can just continue to use `hotBtn`